CHANGELOG.md

```diff
@@ -3,6 +3,184 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v7.0.0
+
+_Mar 20, 2024_
+
+We're excited to [announce the first v7 stable release](https://mui.com/blog/mui-x-v7/)! 🎉🚀
+
+This is now the officially supported major version, where we'll keep rolling out new features, bug fixes, and improvements.
+Migration guides are available with a complete list of the breaking changes:
+
+- [Data Grid](https://mui.com/x/migration/migration-data-grid-v6/)
+- [Date Pickers](https://mui.com/x/migration/migration-pickers-v6/)
+- [Tree View](https://mui.com/x/migration/migration-tree-view-v6/)
+- [Charts](https://mui.com/x/migration/migration-charts-v6/)
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Use renderer interceptor on `DateTimePicker` (#12441) @LukasTy
```

Comment:
You can even remove the "by reusing ...".

- Optimize `DesktopDateTimePicker`: This put me star in the eyes
- by reusing existing view renderers: I'm confused because I'm not that  deep in the interals of the pickers

Reply:
```suggestion
- 🚀 Improve the usage of custom `viewRenderers`  on `DateTimePicker` (#12441) @LukasTy
```